packages/waas/src/intents/base.ts

```diff
@@ -11,9 +11,34 @@ export type SignedIntent<T> = Omit<RawIntent, 'data'> & { data: T }
 const INTENTS_VERSION = 1
 const VERSION = `${INTENTS_VERSION} (Web ${PACKAGE_VERSION})`
 
+let timeDrift: number | undefined
+const timeDriftKey = '@sequence.timeDrift'
+
+function isSessionStorageAvailable() {
+  return typeof window === 'object' && typeof window.sessionStorage === 'object'
+}
+
+export function getTimeDrift() {
+  if (isSessionStorageAvailable()) {
+    const drift = sessionStorage.getItem(timeDriftKey)
```

Comment:
should we use `window.sessionStorage.getItem(..)` to be explicit..? the "window" part that is..?